api/src/routers/reservations.js

```diff
@@ -8,11 +8,38 @@ reservations.get("/", async (req, res) => {
   res.send(reservations);
 });
 
-reservations.post("/", async (req, res) => {
-  const body = req.body;
+reservations.post("/add", async (req, res) => {
+  try {
```

Comment:
similar comment as the one seen in meals, let's send proper error responses for the different errors that you can predict can handle. For example you can add multiple try catch to handle the different knex statements to send helpful error messages :)